tests/test_cli_utils.py

```diff
@@ -0,0 +1,104 @@
+"""CLI Test """
+import logging
+import os
+import unittest
+from pathlib import Path
+
+import pytest
+from click.testing import CliRunner
+
+from pheval.cli_pheval_utils import compare_semsim
+
+
+class TestCommandLineInterface(unittest.TestCase):
```

Comment:
Please add unit tests for methods as well, not just integrations tests for the whole CLI